stress_testing/get.yml

```diff
@@ -0,0 +1,22 @@
+config:
+  target: 'http://localhost:4000'
+  phases:
+    - duration: 5
+      arrivalRate: 1
+      name: test
+    - duration: 60
+      arrivalRate: 10
+      name: low
+    - duration: 60
+      arrivalRate: 100
+      name: medium
+    - duration: 60
+      arrivalRate: 1000
+      name: high
+  processor: './functions.js'
```

Comment:
I am not sure about this but you might have to apply the processor to each phase...? If you conducted the test and this worked, then obviously you dont have to but I have seen it before where the processor is applied for each phase duration.

Reply:
good point, i'll def look into this and adjust if need be